flux-middle/src/global_env.rs

```diff
@@ -107,6 +107,10 @@ impl<'sess, 'tcx> GlobalEnv<'sess, 'tcx> {
         self.queries.check_wf(self, flux_id.into())
     }
 
+    pub fn refparams_of(&self, def_id: impl Into<DefId>) -> QueryResult<Vec<rty::RefineParam>> {
+        self.queries.refine_params_of(self, def_id.into())
+    }
```

Comment:
My point is that we can get rid of the query and make this

```suggestion
    pub fn refparams_of(&self, def_id: impl Into<DefId>) -> QueryResult<List<rty::RefineParam>> {
        Ok(self.generics_of(def_id)?.refine_params.clone())
    }
```